test/regressions/webpack.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     // Helps debugging and build perf.
     // Bundle size is irrelevant for local serving
     minimize: false,
+    concatenateModules: false,
```

Comment:
Why this has to be done?

Reply:
`concatenateModules` rewrites function names, and there is one part of the docs that displays function names, it's the theme object viewer. It produced a visual regression because of this.

Reply:
But why didn't it fail earlier? How is it related to this PR?

Reply:
because before we were concatenating modules that were compiled to Commonjs by babel and i believe those enclosed each module with a closure. Meaning no top level functions and this no need to for renaming. now webpack handles the linking and transformation of module syntax and it does it differently. without a closure, but requiring renaming to guarantee uniqueness.